gufe/transformation.py

```diff
@@ -0,0 +1,51 @@
+from .protocols import Protocol
+
+
+class Transformation:
```

Comment:
Is this meant to be the parent class for all edges?

If so, given all the user stories from F@H alchemy, would it make sense not to call it `Transformation`, are there any cases where we wouldn't be doing a transformation between two distinct alchemical states but still using this parent class?

Reply:
Might try and draft this in now. Basically just a `Transformation` with only one state, not two.